### ISSUE TITLE
Pug/Jade support no longer working after upgrading VS Code to 1.24

### ISSUE THREAD

**Issue by txitguy** [vuejs/vetur#812]
- [x] I have searched through existing issues
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: Window 10
- Vetur version: 0.12.4
- VS Code version: 1.24

## Problem

After updating VS Code to 1.24 there is no longer syntax highlighting for pug/jade.

See Image: https://i.imgur.com/lmKb4ji.png

## Reproducible Case

Create a .vue file in VS Code 1.24 and put the following code:

```vue
<template lang="pug">
  #page
    h1 This is a header
    p This is a body of text with Mark's apostrophe to demonstrate
</template>
<script lang="ts">
import Vue from 'vue'
import Component from 'vue-class-component'

@Component
export default class MyComponent from Vue {}
</script>
```

**Comment by octref** [comment]
https://github.com/Microsoft/vscode/issues/50086 is the upstream issue.

**Comment by octref** [comment]
Fixed in 0.12.5.

**Comment by hxshandle** [comment]
@octref 

I'm sorry I tried 0.12.5 but the issue was remain.

-Platform: Window 10
-Vetur version: 0.12.5
-VS Code version: 1.24.1
